.github/workflows/lint.yml

```diff
@@ -18,4 +18,4 @@ jobs:
       - shell: bash
         run: curl -s -S --retry 3 $BASEDIR/tests/script.sh | bash -
       - run: pip install -r requirements_dev.txt
-      - run: pytest /tmp/test_requirements.py
+      - run: pytest tests -W error
```

Comment:
We typically run an application's tests in a `ci.yml` file: https://ocp-software-handbook.readthedocs.io/en/latest/python/ci.html#python-ci-templates

The `pytest /tmp/test_requirements.py` line is running this test: https://github.com/open-contracting/standard-maintenance-scripts/blob/main/tests/test_requirements.py

That test guarantees that requirements are being managed correctly, as documented here: https://ocp-software-handbook.readthedocs.io/en/latest/python/requirements.html

Reply:
Hello James. Thanks for the documentation, let me take a look and try to fix the rest of the errors I got